File: palindrome.js
exports.palindrome = function(word) {
    let reversed = []
    let allowed = 'abcdefghijklmnopqrstuvwxyz123456789'
    let str1 = word.toString().toLowerCase().split('')
    let str2 = ''
    for (let char of str1) {
        if (allowed.includes(char)) {
            str2 += char
        }
    }
    str2.split('')
    for (let char of str2) {
        if (allowed.includes(char)) {
            reversed.unshift(char)
        }
    }
    let result = reversed.join('')
    return result === str2
};

File: palindrome.py
def palindrome(word):
    string = str(word).lower()
    allowed = 'abcdefghijklmnopqrstuvwxyz123456789'
    string_list = []
    for c in string:
        if c in allowed:
            string_list.append(c)
    string_2 = "".join(string_list)
    test_string = string_2[::-1]
    print(test_string)
    if test_string == string_2:
        return True
    
